web_responsive/static/src/less/main.less

```diff
@@ -4,6 +4,7 @@
 body {
     width: 100%;
     height: 100%;
+    overflow: hidden!important; // Important is to override base template
```

Comment:
You knew this was going to happen... Please change this 😆 

A simple trick for these kind of overriding is:

```
html & {
  [syles...]
}
```

`html body` is more concise than `body` selector, so it will win the fight 😉